### DISCUSSION TITLE
Error in both Chrome and Brave browser while click on the writer icon

### DISCUSSION THREAD

**Discussion by maxmit81** [GorvGoyl/ChatGPT-Writer#25] [Report Bugs and Issues]
this is error i am getting in brave browser .. despite doing the change of settings as mentioned in the trouble shooting section
![image](https://user-images.githubusercontent.com/90784599/213526897-48e8f9f5-2054-480a-ab01-4f3a87265575.png)

and in chrome I get the following error 
![image](https://user-images.githubusercontent.com/90784599/213527008-d802817a-5079-4f83-8c75-e2fbb3b06695.png)

Please help in resolving this issue cause otherwise the extension cant be used.

**Comment by GorvGoyl** [comment]
if you visit chat.openai.com the error should go away.

**Comment by maxmit81** [comment]
Thank you for letting me know. I will check chat.openai.com and let you
know if the error is resolved.


Inserted through the extension .. so working on chrome

On Fri, Jan 20, 2023 at 3:05 PM Gourav Goyal ***@***.***>
wrote:

> if you visit chat.openai.com the error should go away.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GorvGoyl/ChatGPT-Writer/discussions/25#discussioncomment-4735668>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVUUGVZ3AIJOXBR2JHPM56LWTJL4LANCNFSM6AAAAAAUAU34FM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
-

------
Mitesh K. Patel

**Comment by vfonic** [comment]
```
ChatGPT Writer 
{message: 'response is not okay: 400'}
writer.fd84a220.js:formatted:15005 
{domain: 'mail.google.com', OS: 'Mac', browserName: 'Brave', browserVersion: '109.0.0.0'}
writer.fd84a220.js:formatted:15005 
{domain: 'mail.google.com', OS: 'Mac', browserName: 'Brave', browserVersion: '109.0.0.0'}
writer.fd84a220.js:formatted:15004 ChatGPT Writer 
{error: {…}}
error
: 
{detail: 'An error ocurred. Try refreshing your browser.'}
```

I'm having the same error. I have ChatGPT Plus subscription and using Brave browser. I disabled language-based fingerprinting prevention as per instructions.

I refreshed the page and opened chat.openai.com several times and nothing happened.
